Minor fixes for formatting (#37268)
DataDog/documentation: content/en/incident_response/status_pages/_index.md

{{< img src="service_management/status_pages/status_pages_enable_slack.png" alt="Status page settings showing the Enable Slack subscriptions toggle and the Slack App Icon upload" style="width:80%;" >}}

> When enabled, clicking **Subscribe** on the published page opens a modal with **Email** and **Slack** tabs.

Reviewer: I like the change, but one caveat is that customers could have both slack and email, or only slack enabled - so it won't always be both of them

Author: ```suggestion
Click **Subscribe** on the published page to open a modal with a tab for each enabled subscription type.
```

1. Click **Subscribe** and open the **Slack** tab.
1. (Optional) Select **Subscribe to specific services** to choose individual components, or leave it cleared to follow the entire page.
1. Click **Subscribe via Slack**.
   {{< img src="service_management/status_pages/status_pages_slack_subscription_modal.png" alt="Subscribe to Updates modal with the Slack tab selected and a Subscribe via Slack button" style="width:70%;" >}}
1. Authorize the **Datadog Status Pages** app for your workspace and select the channel to receive updates.
   {{< img src="service_management/status_pages/status_pages_slack_oauth.png" alt="Slack authorization screen granting the Datadog Status Pages app access to a workspace and channel" style="width:70%;" >}}

After subscribing, the selected channel receives a welcome message confirming the subscription.

> **Private channels**: The app cannot be added automatically. After subscribing, the channel receives a message prompting you to invite the **Datadog Status Pages** bot. After you invite the bot to the channel, it can post updates. Direct message (DM) channels are not supported. On **internal** status pages, you must be logged in to the same Datadog organization to subscribe.

Reviewer: Minor change: the user installing receives a message prompting you to invite the bot located in the 'messages' tab of the Slack App

<img width="1444" height="146" alt="Image" src="https://github.com/user-attachments/assets/1110c3c5-6b93-4369-b407-cb71d41e7547" />

Author: ```suggestion
**Private channels**: After subscribing, you receive a message in the **Messages** tab of the Slack app prompting you to invite the **Datadog Status Pages** bot to the channel. You must invite the bot before it can post updates. Direct message (DM) channels are not supported. On **internal** status pages, you must be logged in to the same Datadog organization to subscribe.
```
